packages/firebase_storage/firebase_storage/macos/firebase_storage.podspec

```diff
@@ -15,6 +15,24 @@ else
   end
 end
 
+# TODO(Salakar): Remove deployment target check once default Flutter osx minimum updated to 10.12.
+begin
```

Comment:
Feels like firebase_core could define a function for you, this is massively repetitive (there are a few other chunks that are really repetitive now that I think about it). These are correct just inelegant

Reply:
Good idea. I think we should look at this in a follow up PR as would require file requires etc and would need to spend some time making sure it all worked - want to get this fix out given that it's broken currently on current release

Reply:
We may even get away with just having this script in `firebase_core` only rather than all podspecs, but again would need to test it out

Reply:
Indeed - that was the thought - firebase_core facility. I'm a huge fan of the github context many 'create new issue...' feature for this reason, I agree spiraling out in same PR is counterproductive here as this is already working it's way through CI and thus getting testing as is on a build break issue (thus expediency has value)